liquidity/liquidity.go

```diff
@@ -26,6 +30,10 @@ type Config struct {
 	// to loop out swaps.
 	LoopOutRestrictions func(ctx context.Context) (*Restrictions, error)
 
+	// ListSwaps returns the set of swaps that loop has already created.
+	// These swaps may be in a final or pending state.
+	ListSwaps func(ctx context.Context) ([]ExistingSwap, error)
```

Comment:
What is the status on fixing this dependency problem?

Reply:
Turns out that we are ok without a refactor, since we need to use the `Client` functions. There is some logic in the rpc that we will need to duplicate (eg, set sweep addr), but it's quite minimal. Not using the rpc because when we get to dispatching swaps, we need to be able to label them with reserved prefix (to identify autoloops), and that will be validated on rpc level (currently it's in the wrong place, going to move it up)